service/ntfy/ntfy.go

```diff
@@ -0,0 +1,174 @@
+package ntfy
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"strings"
+	"time"
+
+	"github.com/pkg/errors"
+)
+
+// Service allow you to configure Ntfy service.
+type Service struct {
+	client     *http.Client
+	serverURLs []string
+}
+
+func defaultHTTPClient() *http.Client {
+	return &http.Client{
+		Timeout: 5 * time.Second,
+	}
+}
+
+// DefaultServerURL is the default server to use for the Ntfy service.
+const DefaultServerURL = "https://ntfy.sh"
```

Comment:
I don't see a need for this to be exported. We can keep the usage experience a little bit cleaner by making it non-exported.

```suggestion
const defaultServerURL = "https://ntfy.sh"
```